src/game/server/props.cpp

```diff
@@ -1801,10 +1803,11 @@ void CBreakableProp::Break( CBaseEntity *pBreaker, const CTakeDamageInfo &info )
 			}
 		}
 	}
+}
 
-#ifndef HL2MP
-	UTIL_Remove( this );
-#endif
+void CBreakableProp::OnBreak( const Vector &, const AngularImpulse &, CBaseEntity * )
+{
+	Remove();
```

Comment:
Why are we causing the remove to occur so early? We haven't even finished the `Break` invocation and we're already destroying the prop? Would having `EFL_KILLME` be set while the damage handler was still running cause any unexpected behavior?